web/web.bzl

```diff
@@ -171,18 +169,33 @@ def browser(testonly=True, **kwargs):
   browser_alias(testonly=testonly, **kwargs)
 
 
-def web_test(browser, test, data=None, size=None, **kwargs):
+def web_test(browser,
+             test,
+             config=None,
+             data=None,
+             launcher=None,
+             size=None,
+             **kwargs):
   """Wrapper around web_test to correctly set defaults.
   
   Args:
     data
     size: default = "large"
     **kwargs: see web_test in //web/internal:web_test.bzl
   """
+  config = config or str(Label("//web:default_config"))
+  launcher = launcher or str(Label("//go/launcher:main"))
   data = lists.clone(data)
```

Comment:
Rather than doing this, consider doing the following in web_test.bzl:

```python
  return struct(
      # ...
      runfiles=ctx.runfiles(
          files=(ctx.files.browser +
                 ctx.files.config +
                 ctx.files.launcher +
                 ctx.files.test),
          transitive_files=(collect_default_runfiles([ctx.attr.browser,
                                                      ctx.attr.config,
                                                      ctx.attr.launcher,
                                                      ctx.attr.test]) |
                            collect_data_runfiles([ctx.attr.browser,
                                                   ctx.attr.config,
                                                   ctx.attr.launcher,
                                                   ctx.attr.test])),
          collect_data=True,
          collect_default=True))

def collect_default_runfiles(targets):
  """Aggregates default runfiles from targets."""
  data = set()
  for target in targets:
    data += _get_runfiles(target, "default_runfiles")
  return data

def collect_data_runfiles(targets):
  """Aggregates data runfiles from targets."""
  data = set()
  for target in targets:
    data += _get_runfiles(target, "runfiles")
    data += _get_runfiles(target, "data_runfiles")
  return data

def _get_runfiles(target, attribute):
  runfiles = getattr(target, attribute, None)
  if runfiles:
    # Legacy rules create gigantic .jslib files we don't need.
    return [f for f in runfiles.files if not f.path.endswith(".jslib")]
  return []
```

Please note that I'm not 100% sure this is what you want to do. I'm actually talking with Laurent (Master of Skylark) in cl/138992178 hoping he's going to help me grok the runfiles thing, because this review made me realize that I was doing it wrong too.